.github/workflows/build.yaml

```diff
@@ -35,7 +35,7 @@ jobs:
 
         steps:
             - name: Build HT Indexer
-              uses: hathitrust/github_actions/build@v1
+              uses: hathitrust/github_actions/build@ETT-376
```

Comment:
We should make sure this branch is merged and change this back to v1 before merging this PR